### ISSUE TITLE
A11y: toast notifications in extension popup not announced by assistive technologies

### ISSUE THREAD

**Issue by patrickhlauke** [bitwarden/clients#2509]
### Steps To Reproduce

1. Use a screen reader (for instance, Chrome with NVDA on Windows)
2. Toggle some functionality that leads to a toast inside the extension (e.g. copying a note, username, password)
3. A toast appears

### Expected Result

Assistive technologies (e.g. screen readers) should announce the appearance of the toast.

### Actual Result

Assistive technologies don't announce the toast.

### Screenshots or Videos

Demonstration video using Chrome/NVDA. Note that no announcement is made when the toast appears.

https://user-images.githubusercontent.com/895831/163709964-19760126-9d41-4424-89ee-2ea7251050a3.mp4

### Additional Context

The problem itself is upstream in the ngx-toastr component. I filed an issue https://github.com/scttcper/ngx-toastr/issues/933 and a fix PR https://github.com/scttcper/ngx-toastr/pull/934

However, in the meantime, a workaround here (depending on how long it takes for the upstream component to be fixed) would be to explicitly set up a custom container for your toasts, making sure that that container element has `aria-live="polite"`.

### Operating System

Windows

### Operating System Version

10

### Web Browser

Chrome

### Browser Version

100.0.4896.127

### Build Version

1.57.0

**Comment by patrickhlauke** [comment]
And to close the loop, https://github.com/scttcper/ngx-toastr/pull/934 has just been merged

**Comment by patrickhlauke** [comment]
Heads-up @eliykat (and apologies if I keep dragging you into these. let me know if I should just sit patiently instead, don't want to  pester you)

**Comment by djsmith85** [comment]
Awesome upstream-fix @patrickhlauke. 

Unfortunately we'll need to wait to pull those changes in, as we'd first need to bump to `Angular` 13 and then update `ngx-toastr`.

Shouldn't be to far away, from what I've heard.

**Comment by djsmith85** [comment]
@patrickhlauke The bump to Angular 14 and with that the bump of ngx-toastr to 15.0 was done with https://github.com/bitwarden/clients/pull/2948.

This will be available with the `2022.09` release.

**Comment by patrickhlauke** [comment]
confirming that notifications are now announced correctly by screen readers

https://user-images.githubusercontent.com/895831/183534386-6a3c512c-48f0-4141-ae7b-030303bff75b.mp4

arguably, there's a bit of tweaking that could still be done, as they all announced with "Close" at the start (due to the "X" close control). but i'll file this as a separate issue/PR another time.
